bin/worker.js

```diff
@@ -215,9 +212,9 @@ co(function *() {
   config.volumeCache = new VolumeCache(config);
 
   config.gc.on('gc:container:removed', function (container) {
-    container.caches.forEach(co(function* (cacheKey) {
-      yield config.volumeCache.release(cacheKey);
-    }));
+    container.caches.forEach(async (cacheKey) => {
```

Comment:
The way I'm reading it, based on my limited knowledge of co, the original code doesn't wait for all of the promises. Not that it really matters, because none of this is really going to block because they're on a listener function. In fact, you could have it be something like 
```js
config.gc.on('gc:container:removed', () => {
  container.caches.forEach(config.volumeCache.release);
});
```
and it would perform identical to the original code, if that's what you want.